server/order/router/product_router.js

```diff
@@ -9,7 +9,7 @@ const router = express.Router();
 
 router.post('/add', (req, res) => {
```

Comment:
Please don't use req, res. We have to use full name instead of these.
Like request, response.